CMakeLists.txt

```diff
@@ -1127,11 +1131,14 @@ if(HAVE_FILE_OFFSET_BITS)
 endif()
 check_type_size("off_t"  SIZEOF_OFF_T)
 
-# include this header to get the type
-set(CMAKE_REQUIRED_INCLUDES "${CURL_SOURCE_DIR}/include")
-set(CMAKE_EXTRA_INCLUDE_FILES "curl/system.h")
-check_type_size("curl_off_t"  SIZEOF_CURL_OFF_T)
-set(CMAKE_EXTRA_INCLUDE_FILES "")
+
+if(NOT "${SKIP_CURL_CHECKS}")
```

Comment:
does it work to skip checks entirely on the first build?

Reply:
Yes, does it not work for you?

Reply:
nvm, looked like a different kind of check at first
haven't had the time to try building anything yet